libs/utils/sem-color.js

```diff
@@ -10,11 +10,10 @@
  *    '0.1.2-canary.0' => 'cyan'
  */
 
-module.exports = function vc (version) {
-  if (version.match(/\b(alpha|beta|canary)\b/)) {
+module.exports = function vc (version, tag) {
+  if (tag !== 'latest') {
```

Comment:
Should you check for existence here?

```js
if (tag && tag !== 'latest')
```

Otherwise I would think this code would make all npm versions cyan besides latest.

Reply:
Oh yea, probably the below if should be before that one.

```js
if (version.startsWith('0.')) return 'orange';
if (tag && tag !== 'latest') return 'cyan';
return 'blue';
```

That's the whole point that i'm stating in ... ah where it was. :D